tests/users/test_views.py

```diff
@@ -57,7 +58,8 @@ def test_logout_with_next(user, client, logout_url):
 
 
 @pytest.mark.django_db
-def test_register(client, signup_url):
+@patch('apps.users.welcome.WelcomeEmail.dispatch', return_value='/')
```

Comment:
I found a better way to fix the tests and not send a wrong key to the API with every test call.

Reply:
nice!